### PR TITLE
Change division 'Othmer...' to just 'Library' fixes #356

### LINKED ISSUE #356: Try out github projects

### PR DESCRIPTION
**Pull request by jrochkind**

### CODE REVIEW

**Comment by hackmastera** [comment]
This still says it has conflicts. And maybe it would make sense to squash it.

**Comment by jrochkind** [comment]
rebased!

File: spec/factories/generic_works.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@
       additional_title ["Or, There and Back Again q"]
       admin_note    ["This is an admin note"]
       credit_line   ["Courtesy of CHF Collections"]
-      division      "Othmer Library of Chemical History"
+      division      "Othmer"
```
Comment (hackmastera) [inline spec/factories/generic_works.rb]: should be 'library'!

Comment (jrochkind) [inline spec/factories/generic_works.rb]: gah, I already fixed this but messed up in the rebase somehow! ok thanks for the sharp eye. 

Git gets a lot harder with more than one developer involved!

**Comment by jrochkind** [comment]
Don't merge until we figure out how/if we need to sync with data migration.

**Comment by jrochkind** [comment]
okay, master merged in properly. I'm not gonna merge to master, but i _am_ going to deploy to staging and run the bulk task (making 4 different data migration changes) on staging in `screen` right now, before I sign off for the day.

**Comment by hackmastera** [comment]
typo in the PR title; actually refs #365

**Comment by jrochkind** [comment]
oops, sorry! Thanks for the catch! I don't usually reverse digits like that, oops!

**Pull request merged**